internal/gke/create.go

```diff
@@ -243,6 +244,9 @@ func validateCreateRequest(ctx context.Context, client *gkeapi.Service, config *
 	if config.Spec.MaintenanceWindow == nil {
 		return fmt.Errorf(cannotBeNilError, "maintenanceWindow", config.Name)
 	}
+	if config.Spec.Labels == nil {
+		return fmt.Errorf(cannotBeNilError, "labels", config.Name)
+	}
```

Comment:
Labels are required ?

Reply:
Only because of syncing. It can be set to an empty map.

Reply:
Ok, I am assuming if the user doesn't set labels, it is set to empty map in code ?

Reply:
It's the same as for all the other fields, the UI is responsible for setting it.